config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+rZnsNJXszaYhS95au6gMs9jLs6sMReyzm9u93Cfbcd2Os9TekUfdbgfe+uFMVWFdUmpJq84TdTXCNHpUY0KhK1E+vMAtIX5KL0MykVuqMy28LeC3B7KjP2n6JF7O/z4NUBW25MfEx/UTEQWR7oipRa/aIj34NmoD5CJ7IYYvxdoNl877VSqefhlIOofpR/XEu+RLoZT9YT/GUWjYYBLkEKk5GakplKqzTjCOTqkPJchI5K92Tluukb+HFjNLER05JiUiBpopZtVyZDpO/IYG6lOk6wFbY4yskg8zawd9xMnN0m00vd9SForaOiPwznYDTTZ6gu/yiLJzJOvM2UIohCHXlGvrlRzwClyJrJSvwAbJNSHw0A9pNTaTdi+2Rave+yERzaNev4+uHTgN4UYIIf1HqGtcCG0kbR+5--K6DoXjWzJUuAW7hM--GAco5RzmMZNdmxNh4unJ9w==
```

Comment:
create `credentials.yml.enc.example` and push to git, this file you should add to `.gitignore`